theme.js

```diff
@@ -1464,6 +1468,25 @@ export default (variables = defaultThemeVariables) => ({
       right: 0,
       height: NAVIGATION_BAR_HEIGHT,
     },
+    navigationBarImage: {
+      flex: 1,
+      flexGrow: 1,
+      height: NavigationHeader.HEIGHT,
+      left: 0,
+      solidifyOpacityAnimation(driver) {
```

Comment:
why `solidifyOpacity`? This would be applied only if someone gives `animationName="solidifyOpacity"` to navBarImage.